config/routes.rb

```diff
@@ -10,16 +12,20 @@
   resources :carts
   resources :administrator
   resources :about
+  resources :sales
   resources :users do
     resources :orders
   end
 
+  get 'display/:option' => "sales#display", as: "display"
   get 'login' => 'sessions#new'
   get '/payment/processed' => 'checkout#create', as: :get_payment_completed
   get 'admin' => 'administrator#show', as: :dashboard
   get 'admin/foods' => 'administrator#food_index', as: :admin_foods
   get 'admin/orders' => 'administrator#order_index', as: :admin_orders
   get 'admin/users' => 'administrator#user_index', as: :admin_users
+  get 'admin/sales' => 'administrator#sale_index', as: :admin_sales
+  get 'admin/categories' => 'administrator#category_index', as: :admin_categories
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.